commitizen/commands/undo.py

```diff
@@ -0,0 +1,61 @@
+from commitizen import cmd, factory, git, out
+from commitizen.config import BaseConfig
+from commitizen.exceptions import InvalidCommandArgumentError
+
+
+class Undo:
+    """Reset the latest git commit or git tag."""
+
+    def __init__(self, config: BaseConfig, arguments: dict):
+        self.config: BaseConfig = config
+        self.cz = factory.commiter_factory(self.config)
+        self.arguments = arguments
+
+    def _get_bump_command(self):
+        created_tag = git.get_latest_tag()
```

Comment:
Why is this one named as `created_tag` instead of `latest_tag`?

Reply:
The `created_tag` is original name and `get_latest_tag` comes later, I would use `latest_tag` in next commit.